Gruntfile.js

```diff
@@ -212,23 +229,39 @@ module.exports = function (grunt) {
           }
         ]
       }
+    },
+
+    imagemin: {
+      dist: {
+        options: {
+          optimizationLevel: 5
+        },
+        files: [{
+          expand: true,
+          cwd: '<%=pkg.assets%>images',
+          src: ['**/*.{png,jpg,gif}'],
+          dest: '<%=pkg.public%>images'
+        }]
+      }
     }
   });
 
-  grunt.loadNpmTasks("grunt-contrib-watch");
-  grunt.loadNpmTasks("grunt-contrib-connect");
-  grunt.loadNpmTasks("grunt-include-source");
-  grunt.loadNpmTasks("grunt-wiredep");
-  grunt.loadNpmTasks("grunt-bower-install-simple");
-  grunt.loadNpmTasks("grunt-contrib-uglify");
-  grunt.loadNpmTasks("grunt-contrib-concat");
-  grunt.loadNpmTasks("grunt-contrib-cssmin");
-  grunt.loadNpmTasks("grunt-usemin");
-  grunt.loadNpmTasks("grunt-contrib-copy");
-  grunt.loadNpmTasks("grunt-contrib-clean");
-  grunt.loadNpmTasks("grunt-ng-annotate");
-  grunt.loadNpmTasks('grunt-text-replace');
-  grunt.loadNpmTasks('grunt-contrib-rename');
+  require('matchdep').filterDev('grunt-*').forEach(grunt.loadNpmTasks)
```

Comment:
Mixed double and single quotes.<br>Missing semicolon.<br>'require' is not defined.